edx-olive/custom-registration-form
# [SE-4602] feat: add more user-friendly admin UI

## Description

This PR adds a more intuitive admin UI.

Adds the user representation, name, email, is_active and allow_marketing_emails status to list display.

## Screenshots

<img width="1570" alt="Screenshot 2021-09-29 at 15 10 16" src="https://user-images.githubusercontent.com/19173947/135274810-d1df74dd-7dc8-4cea-9e76-65de1470e86d.png">


## Testing instructions

Proofread the code or install the package on an instance and check the admin page.

## Deadline

ASAP.
